packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# 51.17.0 - June 10, 2021
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Add support for SOQL embedded in Apex ([PR #3167](https://github.com/forcedotcom/salesforcedx-vscode/pull/3167))
+
+#### salesforcedx-vscode-core
+
+- Add caching for conflict detection ([PR #3283](https://github.com/forcedotcom/salesforcedx-vscode/pull/3283))
+
+#### salesforcedx-vscode-soql
+
+- Auto-completion improvements for SOQL embedded in Apex ([PR #3228](https://github.com/forcedotcom/salesforcedx-vscode/pull/3228))
```

Comment:
We can exclude this and the one under Apex since these changes are not part of the extension pack yet.